react/features/base/participants/functions.js

```diff
@@ -224,3 +225,28 @@ function _getAllParticipants(stateful) {
             ? stateful
             : toState(stateful)['features/base/participants'] || []);
 }
+
+/**
+ * Returns true if the current local participant is a moderator in the
+ * conference.
+ *
+ * @param {Object|Function} stateful - Object or function that can be resolved
+ * to the Redux state.
+ * @returns {boolean}
+ */
+export function isLocalParticipantModerator(stateful: Object | Function) {
```

Comment:
I expected this to be added as a separate commit which also modified the other features to use it, then use it in this very PR. Let's walk this extra mile, please.